packages/exception/vitest.config.ts

```
@@ -1,32 +1,36 @@
+import { playwright } from '@vitest/browser-playwright';
```

ESLint

[error] Unable to resolve path to module '@vitest/browser-playwright'.

(import-x/no-unresolved)